content/en/blog/_posts/2021-11-02-csi-migration-status.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: blog
+title: "Kubernetes 1.23: Kubernetes In-Tree to CSI Volume Migration Status Update"
+date: 2021-11-03T15:00:00+08:00
+slug: kubernetes-feature-csi-migration-status-update
+---
+
+**Authors:** Jiawei Wang, Software Engineer, Google
+
+The Kubernetes in-tree storage plugin to [Container Storage Interface (CSI)](https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/) migration infrastructure has already been [beta](https://kubernetes.io/blog/2019/12/09/kubernetes-1-17-feature-csi-migration-beta/) since v1.17. CSI migration was introduced as alpha in Kubernetes v1.14.
+
+Since then, Sig Storage and other Sigs are working extra hard to ensure the feature land in great spots. A lot of effort has been put in by the maintainers of each plugin towards making the CSI Migration feature on by default & generally available for their plugins. This blog is intended to give a status update to the feature as well as changes since Kubernetes 1.17 to 1.23. In addition, it will also cover the future roadmap for the CSI migration feature GA for each storage plugins.
+
+## Quick recap: What is CSI Migration and Why?
+
+[Container Storage Interface](https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/) has been GA since Kubernetes v1.13. CSI volume plugin was introduced to make Kubernetes volume plugins more maintainable and truly extensible. It is designed to replace the in-tree storage plugin especially vendor specific plugins for better maintainability(independent release cycle and support) as well as less vulnerability(security/reliability bugs can exist).
```

Comment:
Same here
```suggestion
[Container Storage Interface](/blog/2019/01/15/container-storage-interface-ga/) has been GA since Kubernetes v1.13. CSI volume plugin was introduced to make Kubernetes volume plugins more maintainable and truly extensible. It is designed to replace the in-tree storage plugin especially vendor specific plugins for better maintainability(independent release cycle and support) as well as less vulnerability(security/reliability bugs can exist).
```

Reply:
Done